packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -717,6 +717,76 @@ class PhysicalModel extends SingleChildRenderObjectWidget {
   }
 }
 
+class PhysicalShape extends SingleChildRenderObjectWidget {
+  /// Creates a physical model with a rounded-rectangular clip.
+  ///
+  /// The [color] is required; physical things have a color.
+  ///
+  /// The [shape], [elevation], [color], and [shadowColor] must not be null.
+  const PhysicalShape({
+    Key key,
+    @required this.shape,
+    this.elevation: 0.0,
+    @required this.color,
+    this.shadowColor: const Color(0xFF000000),
+    Widget child,
+    this.textDirection,
+  }) : assert(shape != null),
+       assert(elevation != null),
+       assert(color != null),
+       assert(shadowColor != null),
+       super(key: key, child: child);
+
+  final ShapeBorder shape;
+
+  /// The z-coordinate at which to place this physical object.
+  final double elevation;
+
+  /// The background color.
+  final Color color;
+
+  final Color shadowColor;
```

Comment:
// doc

Reply:
Done, though I don't there's more to add that the field name doesn't say already.